<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>X</string>
   </property>
   <widget class="QPushButton" name="pushButton_setArray">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setArray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSRight">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Left</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setJOG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PowerOn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PowerOn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSRight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Right Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setJOG">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set JOG Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Left Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setABS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGPlus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGMinus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set ABS Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Y</string>
   </property>
   <widget class="QPushButton" name="pushButton_setArray_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setArray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSRight_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSLeft_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Left</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setJOG_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PowerOn_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PowerOn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSRight_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Right Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Home_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setJOG_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set JOG Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSLeft_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Left Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setABS_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGPlus_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGMinus_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABS_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set ABS Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Z</string>
   </property>
   <widget class="QPushButton" name="pushButton_setArray_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setArray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSRight_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ABSLeft_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Left</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setJOG_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PowerOn_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PowerOn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSRight_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Right Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Home_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setJOG_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set JOG Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABSLeft_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set Left Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setABS_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGPlus_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_JOGMinus_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>JOG-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setABS_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set ABS Speed</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
